llvm/include/llvm/Transforms/Instrumentation/SPIRVSanitizerCommonUtils.h

```diff
@@ -26,6 +27,8 @@ constexpr unsigned kSpirOffloadGenericAS = 4;
 
 TargetExtType *getTargetExtType(Type *Ty);
 bool isJointMatrixAccess(Value *V);
+void getFunctionsOfUser(User *User, SmallVectorImpl<Function *> &Functions);
```

Comment:
nit: can you please add a comment for this function?

Reply:
done